### DISCUSSION TITLE
Sending API (Form email Sending)  is very slow

### DISCUSSION THREAD

**Discussion by PrabhuShanmughapriyan** [postalserver/postal#2735] [Help with using Postal]
Hi,
I have done in postal installations on multiple times without any issues. 
But recently I installed in Digital Ocean

**VM configuration**

- 2 vCPUs
- 50 GB memory
- 4 GM RAM

There are no issues in installation and no errors in postal logs. But I found that Email Send API is taking close to 1.5 minutes to complete the request. Admin console is pretty fast asusual but there also send Test message is close to 2 min. 

![Screenshot 2024-01-04 at 10 13 07 AM](https://github.com/postalserver/postal/assets/88906038/6d82c285-53de-41c2-a37b-34990b4bef0d)

In short, emails are sent successfully but both API and Test message send Form is very very slow.

I checked the machine metrics and they are all good.. hardly 2% CPU and not much of RAM usage.

<img width="723" alt="Screenshot 2024-01-04 at 10 13 36 AM" src="https://github.com/postalserver/postal/assets/88906038/9b4b5094-9d6e-49da-af10-ce27d593e02e">

Any help please

**Comment by prabhushan** [comment]
I noticed that Ubuntu has the issue whereas Fedora does not have this issue. Everything is the same...

Is there any restriction from Ubuntu?

**Reply by willpower232** [reply]
Do you need to scale up more workers? Are you using nginx or caddy?

**Reply by PrabhuShanmughapriyan** [reply]
I dont think so.. Because same configuration but when I used Fedora it was fine.. Only Ubuntu had this issue.
